torch/_dynamo/exc.py

```diff
@@ -188,6 +188,13 @@ class IncorrectUsage(Exception):
     pass
 
 
+# TODO: I'm a little uncertain about what error classification we should have
+# for this.  This is potentially a user error, but regressions in
+# specialization in PyTorch proper could also trigger this problem
+class FailOnCacheLimitHit(Exception):
```

Comment:
Maybe subclass of RuntimeError? Not sure indeed.

Reply:
Yeah I really don't know... maybe we should bother the error classification guys some more haha